### PR TITLE
weather changes, issue fixes

### PR DESCRIPTION
**Pull request by buffyuwu**
removes spam from hail damage by removing hail damage.

removes spam from weather updates and also adds a variety via a list so the user's log isn't filled with things like 'rain falls on you.'

### CODE REVIEW

**Comment by Keekenox** [comment]
Noice

**Comment by buffyuwu** [comment]
oh yeah, i forgot, this also fixes non-humans being unable to see the aforementioned weather messages. _robots can see pretty rain fluff text too now._

File: maps/tether/tether_virgo3b.dm [hunk 1]
```diff
@@ -177,11 +177,28 @@ var/datum/planet/virgo3b/planet_virgo3b = null
 		WEATHER_HAIL = 5
 		)
 	observed_message = "It is snowing lightly."
+	effect_message = list(
+		"<I>The gentle breeze lifts tiny particles of falling snow past you.</I>",
+		"<I>Cool wind rushes over your senses as the breeze softly stirs and spirals around you.</I>",
+		"<I>A momentary pause in wind leaves the air still before it finds it's peaceful rythm again.</I>",
+		"<I>Blanketed cold envelops you as the wind carries it's chilled embrace.</I>"
```
Comment (KillianKirilenko) [inline maps/tether/tether_virgo3b.dm]: rhythm, not rythm
its, not it's (possessive vs contraction of it is/it has)

Comment (buffyuwu) [reply maps/tether/tether_virgo3b.dm]: fixed <3

**Comment by buffyuwu** [comment]
this PR will by proxy resolve issue #1594

**Comment by buffyuwu** [comment]
*inhales* this will fix issue #1439 

buffy fix!

**Comment by buffyuwu** [comment]
catsafety

**Comment by KillianKirilenko** [comment]
If you're messing with the title file, maybe change it so that it's not named "Polaris" in the bottom left if you hover over it? I forget where it's defined but if you search for that it should come up.

**Comment by buffyuwu** [comment]
> If you're messing with the title file, maybe change it so that it's not named "Polaris" in the bottom left if you hover over it? I forget where it's defined but if you search for that it should come up.

explain pls

**Comment by KillianKirilenko** [comment]
Yeah sorry I wasn't more specific. And bonus points for fixing the window title.

I would've fixed that myself but it's kinda hard to find that part considering 'VOREStation' is everywhere in the code.

**Comment by buffyuwu** [comment]
it’s in skin.dmf, which won’t return anything to the search! so it’s not really your fault for not finding it; a few months ago i couldn’t either until i put more effort into it lately

**Pull request merged**